### ISSUE TITLE
Restrict Bisq Easy > Offerbook to only offers, and remove chat panel (?)

### ISSUE THREAD

**Issue by suddenwhipvapor** [bisq-network/bisq2#2993]
Wild suggestion, I know.

This was my thought process:
 1. It is really a nuisance having to check every single market chat for new non-offer text messages to moderate/reply to...
 2. If chat features would only be available in the proper chatrooms, this would make my job easier
 3. Users very seldom use offerbook chatroom for chatting, anyway (I know as I scan every currency chat)
 4. Someone who wants to check offerbook chatrooms for normal messages would have a hard time doing that, just like I do (bad UX)
 5. Removing the chat feature from the offerbook would consolidate even further the interaction component of Bisq2, not bad
 6. Once you remove the chat feature from the offerbook, what is even the residual purpose of the chat room itself?
 7. Why not just removing the chat panel under Offerbook altogether, leaving just the Offer list view? That is something new users have complained for quite a while when examining Bisq2, criticizing the chat layout and missing the tabular view from Bisq1
 8. The above would also automatically solve the issue of opening the user details in the offerbook screen when the offerlist is also visible, where the third column goes outside the viewport

**Comment by HenrikJannsen** [comment]
> to check every single market chat for new non-offer text messages to moderate/reply to

I think that is not needed. Ppl who need support can go to support but not expect that support agents find them there. 

But I am not against that idea. We could remove the input field at the bottom and maybe use that for the create offer button (not sure as less visible).

The reason why we did not replicate Bisq 1 offerbook style is because the Bisq Easy concept is based on the idea that its basically a chat room. I think we have to be careful to dilute that concept.

@axpoems What do you think?

**Comment by suddenwhipvapor** [comment]
I understand, my first idea was actually to suggest only the input field be removed, but then the next logic step was "_then what is the chat room good for? the offer list tab would basically replicate the chat with additional filtering and more granular display and be more functional at that than the chatroom_"

**Comment by suddenwhipvapor** [comment]
Of course, I mainly wanted to share my train of thoughts about the matter, born on an admittedly selfish level and expanding from there.
I do not believe getting rid of the chat feature in BE offerbook is a "necessary" step, but I do think it would be an improvement (I only see it used very rarely) yet if support agents/moderators are not required to patrol that section it makes the job more streamlined. Other users can still report offenders to moderators if that's needed.
Monitor chat would be useful even if less so right now, after chat consolidation.
A "chat only" filter would undoubtedly be an improvement for offerbook.
I can still confirm #2923 is still a thing for me, so I suppose with the "chat only" filter that would still be present?

**Comment by suddenwhipvapor** [comment]
> I also believe the monitor would still be a very useful tool. What do you think @HenrikJannsen ?

On second thought, with the many currency-specific offerbook chats, a monitor would actually be very useful, mainly if there was a button near each message to "Jump to this room at this point"

**Comment by HenrikJannsen** [comment]
I guess we can close that
